docs/change-log.md

```diff
@@ -2,6 +2,25 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+
+## Product Update February 24th, 2023
+
+- **New:** Extended MariaDB and MySQL driver inputs with optional parameter `privilege list` that will be applied to the generated user.
+
+- **Fixed:** It is now possible to use browser's native Back button to return to the All Applications screen.
+
+- **Fixed:** Trailing whitespace characters are no longer removed from Shared App Values and Secrets inputs.
+
+- **Fixed:** A draft delta opened through a direct URL didn't render correctly in the UI.
+
+- **Fixed:** Long deployment commends expanding beyond [deployment diff](https://docs.humanitec.com/using-humanitec/work-with-environments/deploy-to-environments/diff-deployments) modal screen. 
+
+- **Fixed:** If a deployment is in progress, the following deployments will no longer be accepted. Instead, they will be rejected with an HTTP 409 error.
```

Comment:
Would something like "Triggering a new deployment while a deployment is in progress will throw an error" work? If possible I'd also add a note on why this is helpful / what pain point is being avoided with this.

Reply:
The problem was that the new deployment was accepted and caused issues if there was one already in progress, so the benefit is they're not accepted but rejected. Is "Triggering a new deployment while a deployment is in progress will throw an error" explaining that?

Reply:
What do you think about "To avoid disruption of ongoing deployments it is no longer possible to trigger a new deployment while a deployment is in progress" ?

Reply:
perfect 😊